integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/xslt/transform_back.xslt

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<!--
+  ~  Copyright (c) 2005-2010, WSO2 Inc. (http://wso2.com) All Rights Reserved.

```

Comment:
2019

Reply:
Changed. Please refer the latest commit.